.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+Please use the following template to submit a pull request, filling
+in at least the "Text for release notes" and/or "Further details".
+Thank You!
+
+# Description
+
+## Text for release notes 
+
+If this pull request should be mentioned in the release notes, 
+please provide its short description matching the style of the GAP
+[Changes manual](https://www.gap-system.org/Manuals/doc/changes/chap0.html).
+
+## Further details
+
+If necessary, please provide further details here.
+
+# Checklist for pull request reviewers
+
+- [ ] proper formatting
+
+If your code contains kernel C code, run clang-format on it; the 
+simplest way is to use `git clang-format`, e.g. like this (don't
+forget to commit the resulting changes):
+
+    git clang-format $(git merge-base master)
+
+- [ ] relevant labels
```

Comment:
```suggestion
- [ ] add relevant labels
```

Reply:
Note that other items in the checklist do not have verbs. For consistency, one could
- leave is as is
- change to `usage of relevant labels`
- add verbs to all items in the checklist

Reply:
Changed to "usage of relevant labels".